lib/Common/Memory/Recycler.inl

```diff
@@ -36,18 +36,22 @@ namespace Memory
 class DummyVTableObject : public FinalizableObject
```

Comment:
I think this should probably match the RecyclerVisitedObject vtable layout?

Reply:
FinalizableObject derives from IRecyclerVisitedObject so it has a matching v-table layout.